fem/bilinearform_ext.cpp

```diff
@@ -903,6 +910,9 @@ void FABilinearFormExtension::Assemble()
             static_cast<const L2ElementRestriction*>(elem_restrict);
          const L2FaceRestriction *restF =
             static_cast<const L2FaceRestriction*>(int_face_restrict_lex);
+         MFEM_VERIFY(
+            !dynamic_cast<const NCL2FaceRestriction*>(int_face_restrict_lex),
+            "Full Assembly not yet supported on NCMesh.");
```

Comment:
One more…

```suggestion
         MFEM_VERIFY(
            fes.Conforming(),
            "Full Assembly not yet supported on NCMesh.");
```

Reply:
done in [4e77bad](https://github.com/mfem/mfem/pull/2840/commits/4e77bad9dd715d0fabd25bd34ed56f949a8079f8)